### PR TITLE
Fix the build system to avoid having dangling IO streams which prevent the main process from closing

### PR DESCRIPTION
**Pull request by victorporof**
This should fix the test suite stalling in the frontend-rewrite-related PRs.

Please see individual commits.

### CODE REVIEW

File: build/task-build-browser.js [hunk 1]
```diff
@@ -3,10 +3,9 @@
 
 import colors from 'colors/safe';
 import fs from 'fs-promise';
+import path from 'path';
 
-import { BROWSER_FRONTENDS_PATH } from './utils/const';
-import { makeDevConfig as dev, makeProdConfig as prod } from './config/webpack.base';
-import { getBuildConfig } from './utils';
+import { BUILD_WEBPACK_CONFIS_PATH, BROWSER_FRONTENDS_PATH } from './utils/const';
```
Comment (jsantell) [inline build/task-build-browser.js]: I think `BUILD_WEBPACK_CONFIS_PATH` is a typo

Comment (victorporof) [inline build/task-build-browser.js]: CONFIGS yes, not CONFIS

Comment (victorporof) [inline build/task-build-browser.js]: Luckily this typo is present everywhere else, so there's no type error.

**Comment by victorporof** [comment]
Fixed typo.

**Pull request merged**